src/gs1/gs1-resolver.controller.ts

```
@@ -0,0 +1,185 @@
+  @Get('/:primaryKey/:primaryValue/:qualifierKey/:qualifierValue')
+  async resolveQualifiedLink(
+    @Param('primaryKey') primaryKey: string,
+    @Param('primaryValue') primaryValue: string,
+    @Param('qualifierKey') qualifierKey: string,
+    @Param('qualifierValue') qualifierValue: string
+  ) {
+    const productId = `${primaryKey}/${primaryValue}/${qualifierKey}/${qualifierValue}`;
+    this.logger.log(`Resolving qualified GS1 Digital Link: ${productId}`);
+    return this.gs1Service.getProduct(productId);
+  }
+} 
```

ESLint

[error] Replace `·` with `⏎`

(prettier/prettier)